tests/check_framework/urls/contains_tuple.py

```diff
@@ -0,0 +1,7 @@
+from django.conf.urls import include, url
+
+urlpatterns = [
+    url('^', include([
```

Comment:
This urlpatterns is this way because it's a copy and edit from one of the others.

We want to test that `check_resolver` checks the url patterns recursively, and doesn't just check the root url conf. Having said that, we probably don't need an `include()` in every `urls.py`. Maybe we should add an explicit test for includes, and simplify the rest of the test urlpatterns.